android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1095,6 +1095,58 @@ public static BlockchainToken createEthereumBlockchainToken(String chainId) {
         return eth;
     }
 
+    public static BlockchainToken makeNetworkAsset(NetworkInfo network) {
+        String logo;
+
+        // TODO: Add missing logos
+        //             case "SOL":
+        //                 logo = "sol.png";
+        //                 break;
+        //             case "FIL":
+        //                 logo = "fil.png";
+        //                 break;
+        //             case network.chainId === BraveWallet.OPTIMISM_MAINNET_CHAIN_ID:
+        //                 logo = "optimism.png";
+        //                 break;
+        //             case network.chainId === BraveWallet.AVALANCHE_MAINNET_CHAIN_ID:
+        //                 logo = "avax.png";
+        //                 break;
+        //             case network.chainId === BraveWallet.FANTOM_MAINNET_CHAIN_ID:
+        //                 logo = "fantom.png";
+        //                 break;
+        //             case network.chainId === BraveWallet.CELO_MAINNET_CHAIN_ID:
+        //                 logo = "celo.png";
+        //                 break;
+        if (network.chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID)) {
+            logo = "eth.png";
+        } else if (network.chainId.equals(BraveWalletConstants.POLYGON_MAINNET_CHAIN_ID)) {
+            logo = "matic.png";
+        } else if (network.chainId.equals(
+                           BraveWalletConstants.BINANCE_SMART_CHAIN_MAINNET_CHAIN_ID)) {
+            logo = "bnb.png";
+        } else if (network.chainId.equals(BraveWalletConstants.POLYGON_MAINNET_CHAIN_ID)) {
```

Comment:
why do we have 4 cases for `POLYGON_MAINNET_CHAIN_ID`?

Reply:
Forgot to clean that up. Fixed now.